js_modules/dagit/packages/core/src/execute/LaunchButton.tsx

```diff
@@ -140,7 +140,7 @@ export const LaunchButtonDropdown = ({
                 position={Position.LEFT}
                 openOnTargetFocus={false}
                 targetTagName="div"
-                content={option.tooltip}
+                content={option.tooltip || ''}
```

Comment:
This looks great to me but I wonder if this causes an empty tooltip to appear when previously a null value would have been no tooltip?

Reply:
Yeah I think empty tooltip content isn't really allowed, I'm not sure what happens in this case but I think it'll all need to be tidied up with the LaunchButton work anyway...